lib/index.ts

```diff
@@ -28,15 +26,14 @@ class PWMetrics {
     view: false,
     expectations: false,
     output: false,
-    // @todo remove when new lighthouse version be released, because -https://github.com/GoogleChrome/lighthouse/pull/1778
```

Comment:
yes what @denar90 said.   we now have cpu throttling **on** as a default
and yeah you can easily disable that.